liquibase-standard/src/main/java/liquibase/change/ColumnConfig.java

```diff
@@ -473,6 +458,30 @@ public Object getValueObject() {
         return null;
     }
 
+
+    /**
+     * Set the default value according to its type.
+     */
+    public ColumnConfig setDefaultValueObject(Object defaultValue) {
+        if (defaultValue != null) {
+            if (defaultValue instanceof Boolean) {
+                setDefaultValueBoolean((Boolean) defaultValue);
+            } else if (defaultValue instanceof Number) {
+                setDefaultValueNumeric(defaultValue.toString());
```

Comment:
This was copied from the constructor, but might make sense to use this:
```
setDefaultValueNumeric((Number) defaultValue);
```

Reply:
```suggestion
                setDefaultValueNumeric((Number) defaultValue);
```

Reply:
Yeap, we can do that.